atom/common/crash_reporter/crash_reporter_win.cc

```diff
@@ -54,6 +54,7 @@ void CrashReporterWin::InitBreakpad(const std::string& product_name,
   if (waiting_event != INVALID_HANDLE_VALUE)
     WaitForSingleObject(waiting_event, 1000);
 
+  breakpad_.reset(nullptr);
```

Comment:
Calling `breakpad_.reset()` is enough, in Chromium codebase it is a custom not to pass parameter when cleaning a pointer.

And can you leave a comment in the code of why it has to be explicitly cleaned?